beta-20161006.md

```diff
@@ -6,7 +6,7 @@ summary: Additions and changes in CockroachDB version beta-20161006.
 
 ## Oct 6, 2016
 
-Although this week's release contains only a few internal changes and doc updates, note that we are preparing to merge numerous features from our `develop` branch to `master` for inclusion in an upcoming release.  
+Although this week's release contains only a few internal changes and doc updates, note that we are preparing to release numerous features currently on our `develop` branch.
```

Comment:
Good point. Can you suggest better language than "we are preparing to release numerous features" though? I'd prefer some specificity. What about something like: "we are preparing to release numerous features that were developed but not released during our recent focus on core stability."